pkg/storage/store.go

```diff
@@ -58,9 +55,6 @@ type Store interface {
 
 	// Delete will delete a record with k key
 	Delete(k string) error
-
-	// Query searches all stored key-value pairs and returns all values that satisfy the query.
-	Query(name string, value string) (StoreIterator, error)
```

Comment:
The PR description should also mention that Query has been removed and the rationale.

Reply:
Done.